.changeset/heavy-lemons-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ima/cli-plugin-scramble-css": minor
+---
+
+Support the exclusion of forbidden classes from scrambled classes
```

Comment:
Can we have more descriptive changelog? Something like "Scrambler no transform class names to potentially harmful class names like `ad`"